.buildkite/scripts/steps/checks/yarn_deduplicate.sh

```diff
@@ -7,4 +7,4 @@ source .buildkite/scripts/common/util.sh
 echo "--- Check yarn.lock for duplicated modules"
 node scripts/yarn_deduplicate
 
-check_for_changed_files 'node scripts/yarn_deduplicate' false 'TO FIX: Run node '"'"'scripts/yarn_deduplicate && yarn kbn bootstrap'"'"' locally, or add an exception to src/dev/yarn_deduplicate/index.ts and then commit the changes and push to your branch'
+check_for_changed_files 'node scripts/yarn_deduplicate' true 'TO FIX: Run node '"'"'scripts/yarn_deduplicate && yarn kbn bootstrap'"'"' locally, or add an exception to src/dev/yarn_deduplicate/index.ts and then commit the changes and push to your branch'
```

Comment:
The script can produces invalid yarn.lock files, usually related to sort order.  It's safest to bootstrap again afterwords.

Reply:
@jbudz, do you think it's enough with changing like 8 to 

```
node scripts/yarn_deduplicate && yarn kbn bootstrap
```

I'm concerned about the extra cost it may have (although it should be a few seconds since we should have bootstrapped earlier, right?).

Should I implement an actual _validate, then run_ approach instead?

Reply:
Could we also add this to the logging of the command?
```suggestion
check_for_changed_files 'node scripts/yarn_deduplicate && yarn kbn bootstrap' true 'TO FIX: Run node '"'"'scripts/yarn_deduplicate && yarn kbn bootstrap'"'"' locally, or add an exception to src/dev/yarn_deduplicate/index.ts and then commit the changes and push to your branch'
```